app/helpers/users_helper.rb

```diff
@@ -35,9 +35,12 @@ def homesite_link(user)
   end
 
   def jabber_link(user)
-    return unless current_account
```

Comment:
Well, I think we should keep this condition for `jabber_link`: this condition hides the link for LinuxFr visitors (anonymous web user).

It avoids to disclose too easily XMPP addresses and so it should avoid some spam on the XMPP accounts.

It's not needed for `mastodon_link`, because Mastodon accounts are already public, isn't it ?

Reply:
I removed the offending line from the commit.